recipe_scrapers/_schemaorg.py

```diff
@@ -309,3 +309,11 @@ def cooking_method(self):
         if cooking_method and isinstance(cooking_method, list):
             cooking_method = cooking_method[0]
         return normalize_string(cooking_method)
+
+    def keywords(self):
+        keywords = self.data.get("keywords")
+        if keywords is None:
+            raise SchemaOrgException("No cooking method data in SchemaOrg")
+        if keywords and isinstance(keywords, list):
+            keywords = keywords[0]
```

Comment:
For consistency I think we should only retain the first keyword in other cases too; perhaps:

```suggestion
        if keywords and isinstance(keywords, list):
            keywords = keywords[0]
        elif keywords and isinstance(keywords, str):
            keywords, _ = keywords.split(",", 1)
```

The content in many of the multi-keyword cases seems either duplicative, or spammy - so retaining only the most significant seems reasonable to me (although whether that's always the first-listed is another question).

Reply:
I suppose the other way to consider this would be to normalize the datatype, but not to add any judgement about the value of the elements contained in the list of keywords.  That would be fine by me too.